kaitai/kaitaistream.cpp

```diff
@@ -565,9 +570,9 @@ std::string kaitai::kstream::bytes_to_str(std::string src, std::string src_enc)
 
     if (cd == (iconv_t) -1) {
         if (errno == EINVAL) {
-            throw std::runtime_error("invalid encoding pair conversion requested");
+            throw std::runtime_error("bytes_to_str: invalid encoding pair conversion requested");
```

Comment:
While we've here, actually, this should be `std::invalid_argument`, I guess, as it's clearly caused by invalid encoding being requested.